mkosi/resources/mkosi-obs/mkosi.postoutput

```diff
@@ -110,6 +110,15 @@ if ((${#DDIS[@]} > 0)); then
     cert-to-efi-sig-list -g "$guid" /usr/src/packages/SOURCES/_projectcert.crt db.esl
     cp db.esl KEK.esl
     cp db.esl PK.esl
+
+    # If there are any additional certs, concatenate them
+    while read -r cert; do
+        rm -f tmp.esl
```

Comment:
Removing this file both at the beginning and end of the loop seems gratuitous, since `tmp.esl` is not used elsewhere in the script.
```suggestion
```